CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ option(USE_BACKEND_BCM_NEXUS_WAYLAND "Whether to enable support for the BCM_NEXU
 option(USE_BACKEND_BCM_RPI "Whether to enable support for the BCM_RPi WPE backend" OFF)
 option(USE_BACKEND_INTEL_CE "Whether to enable support for the Intel CE WPE backend" OFF)
 option(USE_BACKEND_WAYLAND_EGL "Whether to enable support for the wayland-egl WPE backend" OFF)
+option(USE_BACKEND_DIRECTFB "Whether to enable support for the directfb WPE backend" ON)
```

Comment:
Could you put this on OFF, as all the other backend?

Reply:
Updated